scss/fitbounds.scss

```diff
@@ -1,17 +1,11 @@
-
-        .leaflet-retina & {
-            background-image: url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAgCAIAAAD8GO2jAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAAwVpVFh0WE1MOmNvbS5hZG9iZS54bXAAAAAAADw/eHBhY2tldCBiZWdpbj0i77u/IiBpZD0iVzVNME1wQ2VoaUh6cmVTek5UY3prYzlkIj8+IDx4OnhtcG1ldGEgeG1sbnM6eD0iYWRvYmU6bnM6bWV0YS8iIHg6eG1wdGs9IkFkb2JlIFhNUCBDb3JlIDUuNi1jMDY3IDc5LjE1Nzc0NywgMjAxNS8wMy8zMC0yMzo0MDo0MiAgICAgICAgIj4gPHJkZjpSREYgeG1sbnM6cmRmPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5LzAyLzIyLXJkZi1zeW50YXgtbnMjIj4gPHJkZjpEZXNjcmlwdGlvbiByZGY6YWJvdXQ9IiIgeG1sbnM6eG1wTU09Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC9tbS8iIHhtbG5zOnN0UmVmPSJodHRwOi8vbnMuYWRvYmUuY29tL3hhcC8xLjAvc1R5cGUvUmVzb3VyY2VSZWYjIiB4bWxuczp4bXA9Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC8iIHhtcE1NOkRvY3VtZW50SUQ9InhtcC5kaWQ6RjUxNDc5NkRDODJDMTFFNTg1MzFEQjY0MEU0MEY4RDciIHhtcE1NOkluc3RhbmNlSUQ9InhtcC5paWQ6RjUxNDc5NkNDODJDMTFFNTg1MzFEQjY0MEU0MEY4RDciIHhtcDpDcmVhdG9yVG9vbD0iQWRvYmUgUGhvdG9zaG9wICI+IDx4bXBNTTpEZXJpdmVkRnJvbSBzdFJlZjppbnN0YW5jZUlEPSI3MTRDNEJCNEM4MEZBOERDRUNDRkMwNEJERkMyRTE5MyIgc3RSZWY6ZG9jdW1lbnRJRD0iNzE0QzRCQjRDODBGQThEQ0VDQ0ZDMDRCREZDMkUxOTMiLz4gPC9yZGY6RGVzY3JpcHRpb24+IDwvcmRmOlJERj4gPC94OnhtcG1ldGE+IDw/eHBhY2tldCBlbmQ9InIiPz7TUqTmAAACyklEQVR42mJgwAYaWjouXb3+n2gAVAzUwkAkINV00uwgz3Ri7aDEdMJ2EGP6h/dA8OHfP9LtwGP6owf3WhrrXZ2dFOTlebh5uLl5ZOXkbe0dyqtqb9y8TZQduE3/V5CTwcKEL1SjEtI+f/+Nzw5cpt+7ecVAW52YVCctr3Ls1HmcdmA1/dmDW4LcbFjMkpGRk5XFZgvTqQvXsNrBgDUQdVQVkHULikpAPPr3799///7dvnVzYm+XgrQksho+Eenf2IzCYkF3YwWyTje/4LefvmIq+/PjW1p8JLLK6NQCwhb8+vYJOVKtHL3xJ9ykqCBkOx6+eE/AgoXTeuGqGVk5P/34RzBzSQrzwbWU1LahyaKnwU3bd8PZ6dl5vOyMBFNRRWkhnH34wF50aTQLzfS14FJ7j14gpnh4du8aC8wZ0gqqBHzw6fNnKIuZQ1ldlZh8IKmoJi3GD2F//Pz188//KOkXTTUjIxPca39//yaqfP/36/fvv1BXMTExoxqJboGgID9M288bV68SY/7969eevfsC1c7Px8XKiM8CZUVFOHve/HnEWDB73jzkrE4gkjevXIAse+fxK/wx/OPzW3YWhPrGrskEc/Iffg6EDkU1na8/fuOxwMbMAMk9zO8+/yBcVCyfOw3ZEyoaOmcuXMFS3N66bmtuiKxy2vxVRJVFQODlbIsWkiFhUUuXrTh/8dLly5fWrlmVkhjHjJEFGzt6sViAtbj++e2TvqYyA+mgqqEZ3QJcFc6fb59C/b3IsKO8tgHFAvwV8pzpk+VlpHCZJSwqvnjF2sMHd6OJ17d2oliA346/v39u2rA2PTXF3s5OUwMING1t7WLjEhYsWf7pC7Se2LdzEy5/UK1RtH/nZvQ4b+9FsYByO04c2oNmx9df/6jcuDuweyvcKG5h2R9//lO/AXni0F49XR19Q9PTF66iBxEVG6lwABBgAGg9e4vCctzZAAAAAElFTkSuQmCC');
-            background-size: 16px 16px;
-        }
-    }
-}
\ No newline at end of file
+.leaflet-control-fit-bounds {
```

Comment:
Shouldn't this still be scoped within the `.leaflet-dhis2` container?

Reply:
FitBounds control is also used by the split view items which don't have the .leaflet-dhis2, so keeping it like this :-)